pkg/controller/controller.go

```diff
@@ -51,7 +51,7 @@ func (c *RetryController) handleOne(resource interface{}) {
 		c.Retry(resource, err)
 		return
 	}
-	c.RetryManager.Forget(resource)
+	c.Forget(resource)
```

Comment:
was this change intentional, and what does it do? the comment needs updating if so